packages/core/src/index.ts

```diff
@@ -130,6 +130,8 @@ const metric_strategyMaxChannels = create_gauge(
 /// Maximum time to wait for a packet to be pushed to the interaction queue in milliseconds
 const PACKET_QUEUE_TIMEOUT_MILLISECONDS = 15000n
 
+const TICKET_AGGREGATION_TIMEOUT_MILLISECONDS = 10000n
```

Comment:
For large sets this might be too short. Should be configurable.

Reply:
This can be done, once #5486 is merged.

Reply:
The p2p protocol timeout is now configurable.